docs/api-reference-1/frontity.md

```diff
@@ -66,22 +67,27 @@ Frontity exports `fetch` and `URL` with the same API they have in the browser, b
 #### Syntax
 
 ```javascript
-ConnectedComponent = connect(Component);
+ConnectedComponent = connect(Component, options?);
 ```
 
 It's a function that receives a React component an returns the same component but connected to the Frontity state, actions and libraries. Any instance of that component will receive three new props: `state`, `actions` and `libraries`, allowing the component to read the state, manipulate it through actions or use any code other packages have exposed in libraries. Also, that instance will re-render automatically whenever the value of `state` which the component is using is changed.
 
+If you don't want to inject new props in your components, you can use the `injectProps` option set to `false`. That will make your component reactive to changes in the state but without passing new props. For these components to access the state use the [`useConnect`](frontity.md#useConnect) hook.
```

Comment:
Could you clarify what you mean by "without passing new props" ? Which new props is it referring to? Does that mean that the value of the `state` props is never going to be updated in components with `{injectProps: false}` ? 

Edit: I see that you clarify it below, but I would put it here as well 🙂

Reply:
I'll do that, thanks Michal :+1: